runtime/redirector/redirector.c

```diff
@@ -775,8 +781,11 @@ JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args)
 	memset(namedVM, 0, J9_VM_DIR_LENGTH);
 	chooseJVM(args, namedVM, J9_VM_DIR_LENGTH);
 
-	envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
-	if(NULL != envOptions && hasEnvOption(envOptions, "-locateVM")) {
+	envOptions = getenv(ENVVAR_OPENJ9_JAVA_OPTIONS);
+	if (NULL == envOptions) {
+		envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
+	}
+	if (NULL != envOptions && hasEnvOption(envOptions, "-locateVM")) {
```

Comment:
I realize you didn't write this code but since the line is being changed anyway, can you add the extra `(` & `)` to the `NULL != envOptions` clause?

Reply:
No problem.  It was bothering me too.  Fixed.